_posts/2020-06-16-presto-summit-zuora.md

```diff
@@ -13,6 +13,13 @@ Schmiedehausen:
 
 **Using Presto as Query Layer in a Distributed Microservices Architecture**
 
+Update:
+
+We had a great event with lots of questions from the audience, taking us beyond
+the planned time frame. Check out the recording to learn more:
```

Comment:
Is this a verbatim copy from 2020-05-15-state-of-presto.md?

Reply:
yeah .. but thats fine imho.

Reply:
Literally the same thing happened .. we went 15 minutes over with questions.